content/applications/hr/referrals/rewards.rst

```diff
@@ -0,0 +1,76 @@
+=======
+Rewards
+=======
+
+After employees have successfully earned referral points, they can exchange their points by
+purchasing rewards in Odoo's *Referrals* application. Rewards **must** be :ref:`created and
+configured <referrals/create>` before employees can :ref:`redeem points for rewards
+<referrals/redeem>`.
+
+.. _referrals/create:
+
+Create rewards
+==============
+
+Rewards are the only configurations needed when setting up the *Referrals* application. Only users
+with :guilabel:`Administrator` rights for the *Recruitment* application can create or modify
+rewards. To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`.
+Click :guilabel:`New`, and a reward form loads. Enter the following information on the form:
+
+- :guilabel:`Product Name`: enter the name as it should appear for the reward.
+- :guilabel:`Cost`: enter the amount of points required to redeem the reward.
+- :guilabel:`Company`: using the drop-down menu, select the company the reward appears for. If a
+  reward should be used for several companies, each company needs to have a record of the reward,
+  with the company listed on the form. This field **only** appears if in a multi-company
```

Comment:
Just to make sure I'm understanding the explanation... does this mean if a reward should be used for several companies, there needs to a specific (separate) reward form for each company it will be applied to? If so, perhaps, it would be beneficial to consider slightly rewording this explanation to clarify that -- at the moment, it's a bit unclear how "each company needs to have a record of the reward..."

Just something to consider during your revisions...

Reply:
Ahh, OK I see that. But yes, so if there are 3 companies, and they each have a $25 Amazon gift card reward, you will need to create 3 separate $25 Amazon Gift Card Entries, one for each company. I will reword!